### PR TITLE
ACN_CD_CiscoUCS_Connector01

### PR DESCRIPTION
**Pull request by chicduong**
#  Proposed Changes

  - Cisco UCS - Data connector page JSON
  - Cisco UCS - Sample Data JSON
  -

### CODE REVIEW

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "CiscoUCS",
+    "title": "Cisco UCS (Preview)",
```
[the remark this answers is not in the document]
Comment (chicduong) [reply DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json]: changes have been made and resubmitted

File: DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "CiscoUCS",
+    "title": "Cisco UCS (Preview)", 
+    "publisher": "Cisco", 
+    "descriptionMarkdown": "The [Cisco Unified Computing System (UCS)](https://www.cisco.com/c/en/us/products/servers-unified-computing/index.html) connector allows you to easily connect your Cisco UCS logs with Azure Sentinel This gives you more insight into your organization's network and improves your security operation capabilities.", 
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CiscoUCS** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinelgithubparserssymantecvip)",
+    "graphQueries": [ 
+     { 
+         "metricName": "Total data received", 
+         "legend": "Cisco UCS",
```
[the remark this answers is not in the document]
Comment (chicduong) [reply DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json]: changes have been made and resubmitted

File: DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "CiscoUCS",
+    "title": "Cisco UCS (Preview)", 
+    "publisher": "Cisco", 
+    "descriptionMarkdown": "The [Cisco Unified Computing System (UCS)](https://www.cisco.com/c/en/us/products/servers-unified-computing/index.html) connector allows you to easily connect your Cisco UCS logs with Azure Sentinel This gives you more insight into your organization's network and improves your security operation capabilities.", 
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CiscoUCS** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinelgithubparserssymantecvip)",
```
[the remark this answers is not in the document]
Comment (chicduong) [reply DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json]: Parser URL changed to https://aka.ms/sentinelgithubparsersciscoucs as a place holder. @preetikr to create the official URL

File: DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                {
+                    "name": "Cisco UCS",
+                    "description": "must be configured to export logs via Syslog"
+                }
+            ]
+    }, 
+            "instructionSteps": [
+                { 
+                    "title": "", 
+                    "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow the steps](https://aka.ms/sentinelgithubparserssymantecvip) to use the Kusto function alias, **CiscoUCS**",
```
[the remark this answers is not in the document]
Comment (chicduong) [reply DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json]: Parser URL changed to https://aka.ms/sentinelgithubparsersciscoucs as a place holder. @preetikr to create the official URL

File: DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (v-jayakal) [reply DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json]: Please update PR with redirection link : https://aka.ms/sentinel-ciscoucs-function.

File: DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (v-jayakal) [reply DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json]: Please update PR with redirection link : https://aka.ms/sentinel-ciscoucs-function.

File: DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (chicduong) [reply DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json]: links have been updated

File: DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (chicduong) [reply DataConnectors/Cisco UCS/Connector_Syslog_CiscoUCS.json]: links have been updated

**Review by v-jayakal — APPROVED** [review]

**Review by nazang — APPROVED** [review]

**Pull request merged**